lockdown.json

```diff
@@ -134,23 +135,22 @@
     "1.0.5": "07b54ca30286abd1718a0e2a830803efdc9bfa04"
   },
   "browser-pack": {
-    "4.0.1": "7f4ab2b3a11c36a9274141fb7912122fc974e5f6"
+    "4.0.4": "8dae95a20ca43b3fea201faa6cfaa84ff4a0d484"
   },
   "browser-resolve": {
-    "1.8.2": "a00b06c871bee21b50d450ff94c2a423bc787729"
+    "1.9.0": "a3ab60185582063f8317fe48a1380f2ea81425ec"
   },
   "browserify": {
-    "9.0.4": "e58da3d5fe415bafa5734d1cb8e8c9d0ddb27d43"
+    "9.0.3": "f2f742b82ec5631c64b8c98a9788db0017c6517c"
```

Comment:
If you look at the package.json, you can see that we have been asking for `^9.0.3`, which means "anything that is sem-ver-close-enough to 9.0.3. When we did the lock the first time, it got the newest version, 9.0.4. I removed the `^` operator on the version, so it reverted back to 9.0.3.